src/idl/google/protobuf/timestamp.proto

```diff
@@ -0,0 +1,147 @@
+// Protocol Buffers - Google's data interchange format
```

Comment:
This file, `timestamp.proto`, is copied verbatim from the main repo. You should disregard it.